app/server.php

```diff
@@ -7,6 +7,8 @@
  * @author   Taylor Otwell <taylor@laravel.com>
  */
 
+
```

Comment:
Remove these 2 new blank lines from server.php so the pull request is clear of changes to the file.  We never really intended for the file to be updated.  I guess that might have been updated by accident from when we troubleshooted the PHP extension problems.